test/data/script.m

```diff
@@ -0,0 +1 @@
+Print["Hello"];
```

Comment:
It would be awesome if you could get your editor to add newlines at the end of a file.

Reply:
I forgot to format the file.